code/web/src/modules/user/api/actions.js

```diff
@@ -54,106 +58,72 @@ export function login(userCredentials, isLoading = true) {
 
         dispatch({
           type: LOGIN_RESPONSE,
-          error
+          error,
         })
       })
-      .catch(error => {
+      .catch((error) => {
         dispatch({
-            type: LOGIN_REQUEST,
-            isLoading,
-        });
-
-        return axios
-            .post(
-                routeApi,
-                query({
-                    operation: 'userLogin',
-                    variables: userCredentials,
-                    fields: ['user {name, email,role }', 'token'],
-                })
-            )
-            .then((response) => {
-                let error = '';
-
-                if (response.data.errors && response.data.errors.length > 0) {
-                    error = response.data.errors[0].message;
-                } else if (response.data.data.userLogin.token !== '') {
-                    const token = response.data.data.userLogin.token;
-                    const user = response.data.data.userLogin.user;
-
-                    dispatch(setUser(token, user));
-
-                    loginSetUserLocalStorageAndCookie(token, user);
-                }
-
-                dispatch({
-                    type: LOGIN_RESPONSE,
-                    error,
-                });
-            })
-            .catch((error) => {
-                dispatch({
-                    type: LOGIN_RESPONSE,
-                    error: 'Please try again',
-                });
-            });
-    };
+          type: LOGIN_RESPONSE,
+          error: 'Please try again',
+        })
+      })
+  }
```

Comment:
The fully working POST will be on my branch.  I will push it up once this is approved and merged.